app/helpers/application_helper.rb

```diff
@@ -160,7 +160,7 @@ def milliseconds_to_formatted_time( milliseconds )
   # @return [String] time in HH:MM:SS
   def pretty_time( milliseconds )
     duration = milliseconds/1000
-    Time.at(duration).utc.strftime(duration<3600?'%M:%S':'%H:%M:%S')
```

Comment:
This was breaking the way emacs parses the file (emacs was interpreting the rest of the file after this line as a comment).